### PR TITLE
Add stub Vulkan backend

### PR DESCRIPTION
**Pull request by Kangz**
PTAL, this just makes the Vulkan backend a worst copy of the null backend

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -37,6 +37,7 @@ option(NXT_USE_WERROR "Treat warnings as error (useful for CI)" 0)
 # Default values for the backend-enabling options
 set(ENABLE_D3D12 OFF)
 set(ENABLE_METAL OFF)
+set(ENABLE_METAL OFF)
```
Comment (kainino0x) [inline CMakeLists.txt]: `ENABLE_VULKAN`?

File: src/backend/CMakeLists.txt [hunk 2]
```diff
@@ -13,11 +13,11 @@
 # limitations under the License.
 
 set(BACKEND_DIR ${CMAKE_CURRENT_SOURCE_DIR})
+set(D3D12_DIR ${CMAKE_CURRENT_SOURCE_DIR}/d3d12)
 set(METAL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/metal)
 set(NULL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/null)
 set(OPENGL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/opengl)
-set(D3D12_DIR ${CMAKE_CURRENT_SOURCE_DIR}/d3d12)
-set(TESTS_DIR ${CMAKE_CURRENT_SOURCE_DIR}/tests)
```
Comment (kainino0x) [inline src/backend/CMakeLists.txt]: What happened to this line?

Comment (Kangz) [reply src/backend/CMakeLists.txt]: Tests aren't in src/backend anymore, it was unused.

File: CMakeLists.txt (hunk 1 above)
Comment (Kangz) [reply CMakeLists.txt]: Whoops, forgot to remove it, thanks for the catch!

**Comment by Kangz** [comment]
Thanks!

**Pull request merged**